statusHandler/presenter/instanceInfoGetters.go

```diff
@@ -78,26 +80,18 @@ func (psh *PresenterStatusHandler) GetNodeName() string {
 
 // GetTotalRewardsValue will return total value of rewards and how the rewards were increased on every second
 // Rewards estimation will be equal with :
-// (numSignedBlocks - numProposedBlocks) * communityPercentage * Rewards +
-// numProposedBlocks * leaderPercentage * Rewards
+// numSignedBlocks * denomination * Rewards
 func (psh *PresenterStatusHandler) GetTotalRewardsValue() (string, string) {
 	rewardsValue := psh.getBigIntFromStringMetric(core.MetricRewardsValue)
-	communityPercentage := psh.getFloatFromStringMetric(core.MetricCommunityPercentage)
-	leaderPercentage := psh.getFloatFromStringMetric(core.MetricLeaderPercentage)
 	numSignedBlocks := psh.getFromCacheAsUint64(core.MetricCountConsensusAcceptedBlocks)
-	numProposedBlocks := psh.getFromCacheAsUint64(core.MetricCountAcceptedBlocks)
-
-	rewardsCommunityCoefficient := math.Abs(float64(int(numSignedBlocks-numProposedBlocks))) * communityPercentage
-	consensusSignedBlocksRewards := big.NewFloat(rewardsCommunityCoefficient)
-	consensusSignedBlocksRewards.Mul(consensusSignedBlocksRewards, big.NewFloat(0).SetInt(rewardsValue))
 
-	rewardsLeaderCoefficient := float64(numProposedBlocks) * leaderPercentage
-	leaderProposedBlocksRewards := big.NewFloat(rewardsLeaderCoefficient)
-	leaderProposedBlocksRewards.Mul(leaderProposedBlocksRewards, big.NewFloat(0).SetInt(rewardsValue))
+	rewardsCoefficient := float64(numSignedBlocks)
+	totalRewardsFloat := big.NewFloat(rewardsCoefficient)
+	totalRewardsFloat.Mul(totalRewardsFloat, denominationCoefficient)
+	totalRewardsFloat.Mul(totalRewardsFloat, big.NewFloat(0).SetInt(rewardsValue))
 
-	totalRewardsBigFloat := big.NewFloat(0).Add(consensusSignedBlocksRewards, leaderProposedBlocksRewards)
 	totalRewards := new(big.Int)
-	totalRewardsBigFloat.Int(totalRewards)
+	totalRewardsFloat.Int(totalRewards)
 
 	difRewards := big.NewInt(0).Sub(totalRewards, psh.totalRewardsOld)
```

Comment:
what is this?
difRewards = totalRewards - totalRewardsOld
totalRewardsOld = totalRewards
totalRewards = totalRewards - difRewards = old totalRewardsOld = why?

Reply:
Ok so this is needed because i want to display in termui console how rewards are increased on every seconds.
ex : totalRewards = 100
      totalRewardsOld = 90
      You will see in termui console  : 90 + 10 ERD

Reply:
Yes but the last sub on totalRewards value doesn't seems right. Why not saving old totalRewardsOld? You could save it directly as string. In this way you reduce one sub on big.Int.

Reply:
Good point.  Last sub on big int  can be removed.